skycatalogs/catalog_creator.py

```diff
@@ -25,6 +24,9 @@
 from .utils.creator_utils import make_MW_extinction_av, make_MW_extinction_rv
 from .objects.base_object import LSST_BANDS
 from .objects.base_object import ROMAN_BANDS
+from .objects.star_object import StarConfigFragment
+from .objects.galaxy_object import GalaxyConfigFragment
+from .objects.diffsky_object import DiffskyConfigFragment
```

Comment:
`SnanaConfigFragment` and `GaiaConfigFragment` aren't used anywhere?

Reply:
No, that's because skyCatalogs is not responsible for making those catalogs.   Those routines are provided in case someone (or some thing) that does produce them wants to also produce a suitable yaml fragment.